src/app/PageTemplates.php

```diff
@@ -58,14 +58,4 @@ private function services()
                         'placeholder' => trans('backpack::pagemanager.content_placeholder'),
                     ]);
     }
```

Comment:
Is this meant to be part of the PR?

Reply:
I just moved `about_us` to the UniquePages trait.

As these traits get published and need to be edited by developers, this serves as an example how to define your templates/uniquePages.